test/manual/README.md

```diff
@@ -32,7 +28,7 @@ sev-snp-measure --mode snp --vcpus 4 --vcpu-type EPYC-v4 --ovmf $OVMF_CODE --ker
 
 To speed up the verification process of attested TLS, download the ARK and ASK certificates using the CLI tool. The CLI tool will download the certificates under your home directory in the `.cocos` directory.
 ```bash
```

Comment:
Add instructions on building the cli so that it can be referenced. Something like

```diff
+ Note: Make sure you have built the CLI before running the following command. If not, run the following command to build the CLI:
+
+ ```bash
+ make cli
+ ```
```